Breaking changes: Ajustes en PSDs, Layouts y patros

alfonsomartinde opened the pull request:
- Los patros se añaden a OBS como carrusel de imágenes en vez de como fuente de navegador.
- Se ha creado un único PSD para todas las escenas, para mejorar consistencia de diseños.
- Se ha reducido un poco el tamaño del minimapa para que quepan los patros de la parada tal y como tiene diseñado CT
- Se han ajustado un poco los diseños para añadir patrocinadores según diseños de CT.
- Se han modificado un poco el color rojo y el azul para que coincidan con lo diseñado por CT.
- Se han añadido archivos JSON para importar en Win y Mac.
- Ya no hay una cámara para cada caster, sino que se importa la room de VDO entera.
- Se han añadido las fuentes al repositorio.
- Se ha añadido una máscara para Dawe, para usar como filtro en OBS

Commit:
Breaking changes: Ajustes en PSDs, Layouts y patros

- Los patros se añaden a OBS como carrusel de imágenes en vez de como fuente de navegador.
- Se ha creado un único PSD para todas las escenas, para mejorar consistencia de diseños.
- Se ha reducido un poco el tamaño del minimapa para que quepan los patros de la parada tal y como tiene diseñado CT
- Se han ajustado un poco los diseños para añadir patrocinadores según diseños de CT.
- Se han modificado un poco el color rojo y el azul para que coincidan con lo diseñado por CT.
- Se han añadido archivos JSON para importar en Win y Mac.
- Ya no hay una cámara para cada caster, sino que se importa la room de VDO entera.
- Se han añadido las fuentes al repositorio.
- Se ha añadido una máscara para Dawe, para usar como filtro en OBS

latra reviewed (approved):
LGTM

A falta de versión sin patros del CT y version para launcher del TR (para presencial y/o nexo)

Commit:
Banda de patros de CT ahora es una imagen independiente:

- Añadida banda patros CT como imagen, para ponerla como fuente.
- Eliminada banda de patros de los fondos en png.
- Actualizado JSON para escenas con patros
- Añadido JSON para escenas sin patros

alfonsomartinde commented:
> LGTM
> 
> A falta de versión sin patros del CT y version para launcher del TR (para presencial y/o nexo)

Me faltaría la versión para launcher del TR, que no sé cómo es

latra commented:
LGTM, abro issue #2 con ejemplo para TR.  Ya lo haremos.
